docs/en/connector-v2/source/Http.md

```diff
@@ -34,6 +34,10 @@ Used to read data from Http.
 | retry_backoff_multiplier_ms | int    | No       | 100           |
 | retry_backoff_max_ms        | int    | No       | 10000         |
 | common-options              |        | No       | -             |
+| paging.pageNoField          | String | No       | -             |
+| paging.pageNo               | String | No       | -             |
+| paging.pageSizeField        | String | No       | -             |
+| paging.pageSize             | int    | No       | -             |
```

Comment:
I think you only need to add one option `pageing` in this table and its type if `Object` like `schema` option.